### PR TITLE
update aws-subnet-size annotation docs to explain the current behaviour

### PR DESCRIPTION
**Pull request by alex-dabija**
### Please remember to

- [x] Give the PR a meaningful title -- it will be shown in the release notes
- [x] Add (or remove) [user questions](https://github.com/giantswarm/docs/blob/master/CONTRIBUTING.md#front-matter) associated with any updated docs

### CODE REVIEW

**Review by calvix — APPROVED** [review]

**Review by njuettner — APPROVED** [review]

**Review by anvddriesch — APPROVED** [review]

File: src/content/ui-api/management-api/crd/awsmachinedeployments.infrastructure.giantswarm.io.md [hunk 1]
```diff
@@ -565,7 +565,7 @@ spec:
 </div>
 
 <div class="annotation-description">
-<p>This annotation is used for configuring the subnet size of AWSCluster or AWSMachineDeployment. The value is a number that will represent the subnet mask used when creating the subnet. This value must be smaller than 28 due to AWS restrictions.</p>
+<p>This annotation is used to configure the workload cluster's node pool subnet size when used on an AWSMachineDeployment resource. The value is a number that will represent the subnet mask used when creating the subnet. It must be smaller than 28 due to AWS restrictions.</p>
```
Comment (marians) [inline src/content/ui-api/management-api/crd/awsmachinedeployments.infrastructure.giantswarm.io.md:568]: Adding formatting to a string literal.

```suggestion
<p>This annotation is used to configure the workload cluster's node pool subnet size when used on an `AWSMachineDeployment` resource. The value is a number that will represent the subnet mask used when creating the subnet. It must be smaller than 28 due to AWS restrictions.</p>
```

If that should be missing in other CRD names here, please apply accordingly. Thanks!

**Review by marians — CHANGES_REQUESTED** [review]
If you are editing this file manually, that's not necessary. The source of truth for annotation info is apiextensions. If the description is changed there, our automation will pick it up in PRs like https://github.com/giantswarm/docs/pull/909

**Comment by marians** [comment]
FYI: It requires a new release of apiextensions for the description update to get published.

**Comment by alex-dabija** [comment]
> If you are editing this file manually, that's not necessary. The source of truth for annotation info is apiextensions. If the description is changed there, our automation will pick it up in PRs like #909

@marians Thanks! I'll make the change to apiextensions.

**Pull request closed without merging**